package.json

```diff
@@ -4,16 +4,20 @@
   "description": "Chrome extension adding extra features to the Bandcamp experience.",
   "main": "index.js",
   "dependencies": {
+    "webpack": "^4.43.0"
   },
   "devDependencies": {
-    "prettier": "^1.19.1",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.9.0",
-    "eslint-plugin-prettier": "^3.1.2"
+    "eslint-plugin-prettier": "^3.1.2",
+    "idb": "^5.0.3",
+    "prettier": "^1.19.1",
+    "webpack-cli": "^3.3.11"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "npx eslint ./src/"
+    "lint": "npx eslint ./src/",
+    "build": "webpack --config webpack.config.js"
```

Comment:
Since you're using `npx` in the `lint` script, you might consider doing the same here like `npx webpack`. The current command relies on Webpack being globally installed by NPM, and since you list Webpack up in `[dev]Dependencies` it is being installed locally.

It works fine because of prior env setup (you very likely installed webpack globally at some point), so no need to change if you want, but you may find that fresh envs and installs may get stuck here because the README setup/install steps do not list `npm install -g webpack`. 

Tl;dr: consider one of the following:
- Adjust this line to `npx webpack ...`
- Include a global webpack install step in the readme